File: Data/Models/RefundCase.cs
namespace Refundeo.Data.Models
{
    public class RefundCase
    {
        public long Id { get; set; }
        public double Amount { get; set; }
        public double RefundAmount { get; set; }
        public bool IsRequested { get; set; }
        public string CustomerId { get; set; }
        public virtual RefundeoUser Customer { get; set; }
        public string MerchantId { get; set; }
        public RefundeoUser Merchant { get; set; }
        public virtual QRCode QRCode { get; set; }
        public virtual Documentation Documentation { get; set; }
    }
}

File: webapp/src/app/models/refundcase.ts
import {MerchantInfo} from './merchantinfo';
import {CustomerInfo} from './customerinfo';

export class RefundCase {
    id: number;
    amount: number;
    refundAmount: number;
    isRequested: boolean;
    isAccepted: boolean;
    isRejected: boolean;
    receiptNumber: string;
    receiptImage: string;
    vatFormImage: string;
    checked: boolean;
    dateCreated: Date;
    dateRequested: Date;
    merchant?: MerchantInfo;
    customer?: CustomerInfo;
}
